Cargo.toml

```diff
@@ -1,7 +1,15 @@
 [package]
 name = "uefi"
 version = "0.17.0"
-authors = ["Gabriel Majeri <gabriel.majeri6@gmail.com>"]
+authors = [
+    "The Rust OSDev team",
+    "Gabriel Majeri <gabriel.majeri6@gmail.com>",
```

Comment:
I included everyone with more than 500 lines of contributions and an email address. I took the latter from the git log.

Reply:
I personally wouldn't mind just leaving "the Rust OSDev team" everywhere and calling it a day, but I don't know if we need to ask the other people who contributed if they're fine with not being included 😅

Reply:
An even simpler solution here could be to remove the field entirely. It used to be required, but there was an [RFC](https://github.com/rust-lang/rfcs/pull/3052) and it was made optional in cargo 1.53: https://github.com/rust-lang/rfcs/pull/3052. The `authors` info isn't shown on crates.io or docs.rs, so essentially no one will see it except by opening the Cargo.toml file. By dropping it we don't have to figure out arbitrary cutoffs for who is included, or let the list become stale. And in the end our authorship is nicely reflected by the git log, issue tracker, github owners, etc :)

Reply:
> I personally wouldn't mind just leaving "the Rust OSDev team" everywhere and calling it a day,

I prefer this. Are you fine with this, @nicholasbishop ?

Reply:
I mildly prefer removing the field since I consider it vestigial, but I don't feel that strongly about it. Using `the Rust OSDev team` is OK by me :)